prow/plugins/trigger/pull-request_test.go

```diff
@@ -509,6 +529,12 @@ func TestHandlePullRequest(t *testing.T) {
 					t.Errorf("exptected job %s to not be set to complete.", tc.jobToAbort.Name)
 				}
 			}
+			if tc.noIssueLabelsAdded && len(tc.issueLabelsAdded) > 0 {
+				t.Errorf("exptected no issue labels added, but got %v", g.IssueLabelsAdded)
+			}
+			if len(tc.issueLabelsAdded) > 0 && !reflect.DeepEqual(tc.issueLabelsAdded, g.IssueLabelsAdded) {
```

Comment:
The boolean check above seems redudundant if you remove the len check here. Also, please use `cmp` for human readable diffs, i.E. `if diff := cmp.Diff(tc.issueLabelsAdded, g.IssueLabelsAdded); diff != "" { err }`

Reply:
Alright, switched to the cmp.Diff method and changed dependent tests.